CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## UNRELEASED
 
 ### Changed
+- ([#203](https://github.com/demos-europe/demosplan-ui/pull/203)) Move from Functional Component to Template in DpContextualHelp ([@salisdemos](https://github.com/salisdemos))
```

Comment:
I'm unsure if we have to mention internal implementation changes like those in the changelog. As i understand it, a changelog should communicate changed behavior, to enable consumers adapt their usages.

Reply:
Me neither. I thought its worth mentioning it, so one can find errors faster if something strange happens after the update